lib/registry/src/package.rs

```diff
@@ -1,8 +1,17 @@
 use crate::PartialWapmConfig;
+use regex::Regex;
 use std::path::PathBuf;
 use std::{fmt, str::FromStr};
 use url::Url;
 
+const REGEX_FULL: &str = r#"^([a-zA-Z0-9\-_]+)/([a-zA-Z0-9\-_]+)(@([a-zA-Z0-9\.\-_]+*))?$"#;
+const REGEX_PACKAGE: &str = r#"^([a-zA-Z0-9\-_]+)/([a-zA-Z0-9\-_]+)$"#;
```

Comment:
Why we need this regex?

Reply:
`REGEX_PACKAGE` was for validating the package name when uploading (otherwise `name = "foo/bar@version"` would have been valid). Since we now always do it on the backend, it's not needed anymore.

Reply:
Ok, lets remove it then

Reply:
Where REGEX_FULL is being used? Shouldn't `wasmer run` in current master user it? If so, why there is a new regex?

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3432/commits/8e865a241168661521ed252b247e42711c340c20